.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
remove all .idea files, as they are your own Intellij configs and not the project you should add .idea into the gitignore file